### PR TITLE
Fixed use of Double instead of double in Employee

### PR DESCRIPTION
**Pull request by mateenparkar**

### CODE REVIEW

File: src/main/java/org/kainos/ea/cli/DeliveryEmployee.java [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-        super(name, salary, bankNumber, NI);
+    @Override
+    public int getEmployeeID() {
+        return EmployeeID;
+    }
+
+    @Override
+    public void setEmployeeID(int employeeID) {
+        EmployeeID = employeeID;
+    }
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/cli/DeliveryEmployee.java]: this isn't needed, you are already getting this behaviour from the base class

File: src/main/java/org/kainos/ea/cli/SalesEmployee.java [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
-        super(name, salary, bankNumber, NI);
+    @Override
+    public int getEmployeeID() {
+        return EmployeeID;
+    }
+
+    @Override
+    public void setEmployeeID(int employeeID) {
+        EmployeeID = employeeID;
+    }
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/cli/SalesEmployee.java]: this isn't needed, you are already getting this behaviour from the base class

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: src/main/java/org/kainos/ea/cli/DeliveryEmployee.java [hunk 3]
```diff
@@ -1,8 +1,10 @@
 package org.kainos.ea.cli;
 
 public class DeliveryEmployee extends Employee{
-    public DeliveryEmployee(String name, Double salary, String bankNumber, String NI) {
-        super(name, salary, bankNumber, NI);
+
+    private int EmployeeID;
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/cli/DeliveryEmployee.java]: this isn't needed

File: src/main/java/org/kainos/ea/cli/SalesEmployee.java [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+    public void setCommision(double commision) {
         this.commision = commision;
     }
 
     private Double commision;
+    
 
-    public SalesEmployee(String name, Double salary, String bankNumber, String NI,Double commission) {
-        super(name, salary, bankNumber, NI);
+    private int EmployeeID;
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/cli/SalesEmployee.java]: this isn't needed

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Review by CallumE74 — APPROVED** [review]

**Pull request merged**